src/awscr-s3/http.cr

```diff
@@ -7,20 +7,20 @@ module Awscr::S3
     @[Deprecated("Use `Http.new(signer, endpoint)` instead")]
```

Comment:
Should this message be updated with a new symantic: "Http.new(signer, endpoint, factory)"?